content/concept-xrp-ledger-intro.md

```diff
@@ -0,0 +1,88 @@
+# XRP Ledger Introduction
+
+The **XRP Ledger** is a decentralized cryptocurrency network. Like all such networks, it's powered by a peer-to-peer server and it enables anyone to send and receive value within the rules of the network. The XRP Ledger is the home of **XRP**, a cryptocurrency which was designed by its creators to bridge the many different currencies preferred by people worldwide. Ripple, the company that stewards development of the XRP Ledger, expects XRP to be a key part of enabling the "Internet of Value": a world in which money moves the way information does today.
+
+## The Best Digital Asset
```

Comment:
Same question as above -- if we want to address it -- do we want to refer to XRP as a digital asset or a cryptocurrency? Or is it both - and we refer to it as a digital asset in some contexts and as a cryptocurrency in others? What are those contexts?